data/remnant missions.txt

```diff
@@ -493,5 +598,761 @@ mission "Remnant: Technology Available"
+			`	Dawn does some follow-up scans of idle Quarg ships, and they take some time to make sure all their pictures are well focused and accurate. They let you know that they will meet you back in the spaceport bar.`
+		payment 50000
+
+mission "Remnant: Expanded Horizons Quarg 3"
+	name "Expanded Horizons Return"
+	description "Take the Remnant researchers to <planet>"
+	passengers 3
+	blocked `This mission requires bunks for three passengers.`
+	source "Wayfarer"
+	destination
+		government "Remnant"
+	to offer
+		has "Remnant: Expanded Horizons Quarg 2: done"
+	on offer
+		conversation
+			`Back in the spaceport bar you wait around for a couple hours. Just when you were about to go find them, the trio walk in looking cheerful.`
+			`	"OK, I think we are ready to go. Can you take us to <planet>?"`
+			choice
+				`Yes`
+					goto ReturnTrip
+				`Not yet`
+					defer
+			label "ReturnTrip"
+			`	"Great. We will be waiting on your ship."`
+				accept
+	on complete
+		payment 50000
+		conversation
+			`Once back in the privacy of <ship>, the trio are a blur of motion as they chatter back and forth with their signs. Apparently this will be one of the biggest boosts to their research since the first time they salvaged the gear off a Korath raider.`
+			`	They eventually slow down their signs a bit and explain that up until the time the Remnant left Human space, the Quarg had been reluctant to let anyone conduct detailed scans of their ships, let alone the close contact the engineers on Tarazed seem to enjoy. One of her fellow researchers picks up the story:`
+			`	"Our historical records just have a few pictures and videos of Quarg ships, along with a basic scan that doesn't provide more than the general outline of the ship. These scans," he taps the databanks "actually picked up detailed structural layouts and even identified a number of key components. It isn't nearly enough information to be able to duplicate them, but it is enough that we can start trying to figure out how they work.`
+			`	Upon arriving at <planet> they quickly thank you, and literally run back to their offices to finish prepping their material to publish and distribute.`
+
+mission "Remnant: Follow-up on Terminus Researchers"
+	name "Check up on Terminus Researchers"
+	description "You offered to checkup on the wormhole researchers"
+	source
+		government Remnant
+	destination Bounty
+	to offer
+		has "Remnant: Learning Sign 2: done"
+		has "Terminus exploration: done"
+		random < 50
+	on offer
+		conversation
+			`Looking around at the Remnant Spaceport, it suddenly strikes you how amazing this is. You never would have imagined that you would end up here back when you earned your pilot's license. It seems so long ago that your biggest concern was deciding whether to pay off your loan by ferrying cargo, doing courier jobs, or escorting merchants. (and on your darker days - dreaming of becoming a pirate).`
```

Comment:
"Spaceport" doesn't need to be capitalized

Reply:
Done.